### ISSUE TITLE
Critical Vulnerabilities wrongly reported on centos:7

### ISSUE THREAD

**Issue by PascalLaporte** [aquasecurity/trivy#319]
**Description**

Scanning centos:7
With: `trivy --severity=CRITICAL  --debug centos:7`

We get the result:
```
+----------------+------------------+----------+-------------------+---------------+--------------------------------+
|    LIBRARY     | VULNERABILITY ID | SEVERITY | INSTALLED VERSION | FIXED VERSION |             TITLE              |
+----------------+------------------+----------+-------------------+---------------+--------------------------------+
| bash           | CVE-2014-6277    | CRITICAL | 4.2.46-31.el7     |               | bash: uninitialized here       |
|                |                  |          |                   |               | document closing delimiter     |
|                |                  |          |                   |               | pointer use                    |
+                +------------------+          +                   +---------------+--------------------------------+
|                | CVE-2014-6278    |          |                   |               | bash: incorrect parsing of     |
|                |                  |          |                   |               | function definitions with      |
|                |                  |          |                   |               | nested command substitutions   |
+----------------+------------------+          +-------------------+---------------+--------------------------------+
(...)
```

**What did you expect to happen?**
CVE-2014-6277 and CVE-2014-6278 shouldn't be there...

**What happened instead?**
CVE-2014-6277 and CVE-2014-6278 or wrongly present in the report...

**Output of run with `-debug`:**

```
2019-12-13T16:08:40.532-0500	DEBUG	cache dir:  /Users/plaporte/Library/Caches/trivy
2019-12-13T16:08:40.533-0500	DEBUG	DB update was skipped because DB is the latest
2019-12-13T16:08:40.533-0500	DEBUG	Vulnerability type:  [os library]
2019-12-13T16:08:44.530-0500	DEBUG	OS family: centos, OS version: 7.6.1810
2019-12-13T16:08:45.052-0500	DEBUG	the number of packages: 146
2019-12-13T16:08:45.052-0500	DEBUG	the number of packages from commands: 0
2019-12-13T16:08:45.052-0500	DEBUG	the number of packages: 146
2019-12-13T16:08:45.052-0500	INFO	Detecting RHEL/CentOS vulnerabilities...
2019-12-13T16:08:45.052-0500	DEBUG	redhat: os version: 7
2019-12-13T16:08:45.052-0500	DEBUG	redhat: the number of packages: 146

centos:7 (centos 7.6.1810)
==========================
Total: 7 (UNKNOWN: 0, LOW: 0, MEDIUM: 0, HIGH: 0, CRITICAL: 7)
...
```

**Output of `trivy -v`:**

```
trivy version 0.2.1
```

**Additional details (base image name, container registry info...):**

This is only an exemple of wrongly reported vulnerability.

It seem that the db is lacking a lot of fixed cve from redhat...
At the current it would be impossible to rely on such result ....

Trivy is supposed to be more accurate than Clair or Anchore. They both doesn't reporte these CVE.

**Comment by knqyf263** [comment]
You need to read README and Red Hat security advisory.
https://github.com/aquasecurity/trivy#ignore-unfixed-vulnerabilities

CVE-2014-6277 and CVE-2014-6278 affect bash on all Red Hat versions and are not provided a patch.
https://access.redhat.com/security/cve/cve-2014-6277
https://access.redhat.com/security/cve/cve-2014-6278

In short, they are not false positives. If you want to ignore them, you can use `--ignore-unfixed` option.

Thanks.

**Comment by samifruit514** [comment]
Hello,

According to this page: https://wiki.centos.org/Security/Shellshock, this CVE would be fixed starting from bash version 4.2.45

Am I right?
